twitter.html

```diff
@@ -0,0 +1,568 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Twitter</title>
+    <link rel="stylesheet" href="style.css">
+
+</head>
+
+<body>
+
+    <section class="grid">
+        <section class="left">
+            <div class="left-menu">
+                <div>
+                    <img src="/twitter-color-icon.svg" alt="twitter-logo" id="logo">
+                </div>
+
+                <div class="home-button two"><svg xmlns="http://www.w3.org/2000/svg"
+                        shape-rendering="geometricPrecision" text-rendering="geometricPrecision"
+                        image-rendering="optimizeQuality" fill-rule="evenodd" clip-rule="evenodd"
+                        viewBox="0 0 512 474.882">
+                        <path fill="#E15241"
+                            d="M77.867 305.714L246.59 136.93c8.908-8.908 10.037-9.285 19.279-.007l168.248 168.791.013 169.168H309.055V360.644c0-7.722-6.322-14.044-14.044-14.044h-78.022c-7.723 0-14.045 6.322-14.045 14.044v114.238H77.867V305.714zM250.715 3.663L0 239.075l61.836 32.472L238.938 92.976c15.13-15.264 15.306-15.598 30.367-.676l180.858 179.247L512 239.075 260.907 3.308c-4.509-4.232-5.138-4.706-10.192.355z" />
+                    </svg>
```

Comment:
I would suggest that it would be cleaner to use img and if you need to recolor the svg then apply a filter to it in your css